apps/widget/src/components/BaseCardPerkERC20.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-    <b-card class="m-2" :class="{ 'card-promoted': perk.isPromoted }">
+    <b-card class="m-2" v-bind:class="[perk.isPromoted ? 'card-promoted' : '']">
```

Comment:
@valeriagrazzini Any particular reason for changing this syntax away from our regular patterns?

Reply:
This was not an issue with the class binding, but the public perks list endpoint not returning the isPromoted flag.

Reply:
yes I see, it's because the sdk it's not updated yet.